appinventor/appengine/src/com/google/appinventor/client/editor/simple/components/CSVFileChangeListener.java

```diff
@@ -0,0 +1,19 @@
+package com.google.appinventor.client.editor.simple.components;
+
+/**
+ * Listener interface for receiving CSVFile column change events.
+ *
+ * Classes referencing a CSVFile source should implement this
+ * interface to receive events when the Source file of the CSVFile
+ * is changed, and the columns are changed as a result.
+ *
+ * Triggering events on property change for the Source are not
```

Comment:
Change last paragraph to highlight asynchronous behaviour (but not specifically focus on property change)